src/test/java/uk/ac/ebi/eva/pipeline/io/readers/AggregatedVcfReaderTest.java

```diff
@@ -1,13 +1,13 @@
 package uk.ac.ebi.eva.pipeline.io.readers;
```

Comment:
Copyright declaration.

Can we have some tests for EVS and EXAC aggregation types?